.github/workflows/build.yaml

```diff
@@ -40,9 +40,10 @@ jobs:
      - name: Add repos
        run: |
          sudo add-apt-repository 'deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-16 main'
-         wget -q -O - https://dl.openfoam.com/add-debian-repo.sh | sudo bash
```

Comment:
Why doesn't this work

Reply:
I dont know  :/ . I would fix it like this for now. If we find a better way we can use wget again. I assume esi changed something on their side, since it suddenly stopped working.